app/controllers/hbx_admin_controller.rb

```diff
@@ -9,7 +9,7 @@ def registry
   end
 
   def edit_aptc_csr
-    raise NotAuthorizedError if !current_user.has_hbx_staff_role?
+    # raise NotAuthorizedError if !current_user.has_hbx_staff_role?
```

Comment:
Why are we removing this?

Reply:
@RyanEddyIC I see you resolved this conversation - can we make sure questions are answered here on GitHub?